torch/csrc/profiler/python/init.cpp

```diff
@@ -293,6 +294,7 @@ void initPythonBindings(PyObject* module) {
   m.def(
       "_disable_execution_graph_observer",
       &torch::profiler::impl::disableExecutionGraphObserver);
+  m.def("_initialize_kineto", &torch::profiler::impl::kineto::initializeKineto);
```

Comment:
Why not just do this in C++ during runtime init? (I just did it through Python because it was expedient.)

Reply:
thanks for the suggestion - also, I think I will try to move this into lazy cuda initialization (the ROCM & windows tests are failing because this is somehow triggering cuda init during `import torch`)